provisioner/roles/manage_ec2_instances/tasks/instances_security_automation.yml

```diff
@@ -1,168 +1,164 @@
-- name: set node types
-  set_fact:
-    node1_node: "{{ ec2_lab_node_types['node1'] }}"
-    node2_node: "{{ ec2_lab_node_types['node2'] }}"
-    node3_node: "{{ ec2_lab_node_types['node3'] }}"
-  when: not networking
-
-- name: find ami for node1
+- name: find ami for splunk
   ec2_ami_facts:
-    region: "{{ ec2_region }}"
-    owners: "{{ ec2_info[node1_node].owners }}"
+    region: "us-east-1" # I don't know why but that seems to be the only place it's available
```

Comment:
If this is the case we need to document that in the example yml for the provisioning.

Reply:
@liquidat which example? the `sample_workshops` or $other?